probe/endpoint/ebpf.go

```diff
@@ -47,50 +40,97 @@ func NewEbpfTracker(bccProgramPath string) *EbpfTracker {
 
 	stderr, err := cmd.StderrPipe()
 	if err != nil {
-		log.Errorf("bcc error: %v", err)
+		log.Errorf("EbpfTracker error: %v", err)
 		return nil
 	}
-	go logPipe("bcc stderr:", stderr)
+	go logPipe("EbpfTracker stderr:", stderr)
 
 	tracker := &EbpfTracker{
-		Cmd: cmd,
+		cmd:         cmd,
+		activeFlows: map[string]ebpfConnection{},
 	}
 	go tracker.run()
 	return tracker
 }
 
+func (t *EbpfTracker) handleFlow(eventStr string, tuple fourTuple, pid int, netns string) {
+	t.Lock()
+	defer t.Unlock()
+
+	switch eventStr {
+	case "connect":
+		log.Infof("EbpfTracker: connect: %s pid=%v", tuple.String(), pid)
+		conn := ebpfConnection{
+			outgoing: true,
+			tuple:    tuple,
+			pid:      pid,
+			netns:    netns,
+		}
+		t.activeFlows[tuple.String()] = conn
+	case "accept":
+		log.Infof("EbpfTracker: accept: %s pid=%v", tuple.String(), pid)
+		conn := ebpfConnection{
+			outgoing: true,
+			tuple:    tuple,
+			pid:      pid,
+			netns:    netns,
+		}
+		t.activeFlows[tuple.String()] = conn
+	case "close":
+		log.Infof("EbpfTracker: close: %s pid=%v", tuple.String(), pid)
+		if deadConn, ok := t.activeFlows[tuple.String()]; ok {
+			delete(t.activeFlows, tuple.String())
+			t.bufferedFlows = append(t.bufferedFlows, deadConn)
```

Comment:
Why do we need to add closed connections to the `bufferedFlows`? Isn't the buffer the state of the previous iteration?